src/components/elements/Table.js

```diff
@@ -40,24 +40,43 @@ const TableStyle = styled.table`
     padding-bottom: 10px;
   }
 
-  & tr:nth-child(even) td {
-    background-color: ${greyScale(9)};
-  }
 `;
 
-export const TableCell = ({ children, header }) => (
+export const TableCell = ({ children, header, }) => (
   <>{header ? <th>{children}</th> : <td>{children}</td>}</>
 );
 
-export const TableBody = styled.tbody``;
+export const TableSpanCell = styled.td`
+  grid-column: 1 / span 5;
+`
+
+export const TableRightAlignCell = styled.td`
+  text-align: right;
+  margin-right: 50px;
+`
+
+export const TableBody = ({children}) => (
+  <tbody>{children}</tbody>
+);
 
 export const TableHeader = ({ children }) => (
   <thead>
     <tr>{children}</tr>
   </thead>
 );
 
-export const TableRow = styled.tr``;
+export const TableRow = styled.tr`
+  &>td {
+  ${props => props.highlighted ? `
+  background-color: rgb(255, 245, 217);
+  ` : `background-color: ${props.backgroundColor}`}
+  }
+  cursor: pointer;
+
+  input[type="checkbox"] {
+    cursor: pointer;
+  }
```

Comment:
Is this style block doing anything?

Reply:
Yeah, it makes the cursor a pointer when hovering the checkbox. Idk why it's needed but it is.